openstack/neutron/values.yaml

```diff
@@ -7,7 +7,7 @@ osprofiler: {}
 
 global:
   dbUser: neutron
-  dbPassword: DEFINED-IN-REGION-CHART
+  dbPassword: null
```

Comment:
Have you tested that this actually works and fails? I would leave those values completly away. It is common practice in helm charts to have required values as a comment in the values file. E.g. https://github.com/helm/charts/blob/master/stable/postgresql/values.yaml#L80